### ISSUE TITLE
taxonRank categories are not standardised

### ISSUE THREAD

**Issue by stijnvanhoey** [inbo/data-publication#122]
I did a count of the different taxonrank fields, checking the current names in the taxonrank column

   taxonRank | kingdom |  count
------------------------| ----------------- | -----------------
Cultivar | Plantae | 	7
Form | Plantae | 3
Generic  hybrid | Plantae | 3
Genus | Plantae | 26410
Nothosubspecies	 | Plantae | 6
Species | Plantae | 3545162
Species aggregate | Plantae | 2489
Species group | Plantae | 22731
Species hybrid | Plantae | 15259
Subspecies | Plantae | 155616
Variety | Plantae | 12024

Is there a more rigid name convention or vocabulary that we could adopt?

**Comment by peterdesmet** [comment]
Yes, we should use this vocabulary: http://rs.gbif.org/vocabulary/gbif/rank_2015-04-24.xml It uses lowercase names + no spaces. Here's a mapping of what it should be:

* [ ] Cultivar → `cultivar` 	
* [ ] Form → `form`
* [ ] Generic  hybrid → not part of vocab, would like to see examples
* [ ] Genus → `genus`
* [ ] Nothosubspecies → would like to see examples
* [ ] Species → `species`
* [ ] Species aggregate → would have to see examples
* [ ] Species group → would have to see examples
* [ ] Species hybrid → would have to see examples
* [ ] Subspecies → `subspecies`
* [ ] Variety → `variety`

@stijnvanhoey can you provide some examples for the ones above?
